### PR TITLE
Front search profile

### PR DESCRIPTION
**Pull request by lambrette**
Search profile component

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -107,4 +114,9 @@ button {
     padding: 2rem;
   }
 }
+
+.search {
+  position: absolute;
+  z-index: 1000;
```
Comment (Crysicia) [inline frontend/src/App.vue:120]: Is this really necessary ?

Comment (lambrette) [reply frontend/src/App.vue:120]: Yes to display the user above the rest of the content

**Review by Crysicia — COMMENTED** [review]
Good

File: frontend/src/App.vue [hunk 2]
```diff
@@ -70,7 +74,10 @@ export default {
     unsubscribeAndDisconnectStatusSocket() {
       this.unsubscribeStatusSocket();
       this.disconnectStatusSocket();
-    }
+    },
+    onUserSelected(user) {
+      this.$router.push(`/profile/${user.nickname}`);
+  }
   }
```
[the remark this answers is not in the document]
Comment (gmorange42) [reply frontend/src/App.vue:81]: Format :p

**Pull request merged**